test/environmentSettings.ts

```diff
@@ -73,6 +78,14 @@ export class EnvironmentSettings {
       EnvironmentSettings._instance._throttleFactor =
         parseInt(process.env.THROTTLE_FACTOR!) ||
         EnvironmentSettings._instance._throttleFactor;
+      if (process.env.WITH_TEST_PROJECT) {
+        EnvironmentSettings._instance._sfdxProjectHome = join(
+          __dirname,
+          '..',
+          '..',
+          process.env.WITH_TEST_PROJECT
```

Comment:
Since this is just the name of the test project (apex-perf-project), `TEST_PROJECT_NAME` or `SFDX_TEST_PROJECT_NAME` seems more clear.

Reply:
I like TEST_PROJECT_NAME, will change